From: Anthony Sottile <asottile@umich.edu>
Date: Sat, 30 Sep 2023 13:16:33 -0400
Subject: test-no-random-order

---
 Lib/test/libregrtest/cmdline.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Lib/test/libregrtest/cmdline.py b/Lib/test/libregrtest/cmdline.py
index 07681d7..e70efaf 100644
--- a/Lib/test/libregrtest/cmdline.py
+++ b/Lib/test/libregrtest/cmdline.py
@@ -456,7 +456,6 @@ def _parse_args(args, **kwargs):
         #   -j0 --randomize --fail-env-changed --rerun --slowest --verbose3
         if ns.use_mp is None:
             ns.use_mp = 0
-        ns.randomize = True
         ns.fail_env_changed = True
         if ns.python is None:
             ns.rerun = True
